### PR TITLE
Enable Hockey App logging for UWP

### PR DESCRIPTION
**Pull request by alinapopa**

### CODE REVIEW

File: src/MobileApps/MyDriving/MyDriving.UWP/project.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿{
   "dependencies": {
     "Acr.UserDialogs": "4.3.4",
-    "HockeySDK.UWP": "3.1.1-build59744",
+    "HockeySDK.UWP": "4.0.0",
```
Comment (VaijanathA) [inline src/MobileApps/MyDriving/MyDriving.UWP/project.json]: Looks good for me. Were you planning to update the version of Android and IOS versions of HockeyApp too ?

Comment (alinapopa) [inline src/MobileApps/MyDriving/MyDriving.UWP/project.json]: I have added the commits with HockeySDK update

Comment (VaijanathA) [inline src/MobileApps/MyDriving/MyDriving.UWP/project.json]: Do we have to update Android too ? Looks good to me.

Comment (alinapopa) [inline src/MobileApps/MyDriving/MyDriving.UWP/project.json]: Android is updated too (commit a9e784b240dad351a3f8e4eb576456bbfd10bbaf )

**Pull request merged**